### DISCUSSION TITLE
NameError: name 'errno' isn't defined

### DISCUSSION THREAD

**Discussion by ghost** [micropython/micropython#12452] [Using MicroPython]
Hey, `raise OSError(errno.ETIMEDOUT)` is working fine when I test it in the REPL or in an .py file, but is raising that error when ran in an .mpy file. I'm using mpy-cross 1.20.0 to compile the file. What could be happening and how can I fix it?
Thanks in advance,

**Comment by robert-hh** [comment]
It would be helpful if you show the script that fails. I assume that when working in REPL, `errno` is defined e.g. as global. IDE's like Thonny could do that without you knowing it.

**Comment by robert-hh** [comment]
I'll transfer this to discussions. It does not look like a MicroPython issue.

**Comment by robert-hh** [comment]
Opened again in Issues, as inconsistency.
